app/common/model/src/main/java/io/syndesis/common/model/WithVersion.java

```diff
@@ -22,4 +22,11 @@
     default int getVersion() {
         return 1;
     }
+
+    /**
+     * Marker interface to indicate that the revision should be automatically updated
+     * by the data manager
+     */
+    interface AutoUpdatable {
```

Comment:
Naming things is hard, this is a good name, might `Current` be a better one? So `WithVersion.Current`?

Reply:
I can do that, but after PTO :)

Reply:
Or `Latest`, not insisting on this really...